javascript-primitives-and-variables/main.js

```diff
@@ -0,0 +1,16 @@
+var fullName = 'Thomas Tahk';
+var isCool = false;
+var totalPets = 1;
+var unknown = null;
+var never;
+
+console.log('value of fullName: ' + fullName);
```

Comment:
you want to use `,` rather than `+` because you don't want to concatenate the values to the string labels. you want separate arguments in console log.